tests/test_coordinate_data.py

```diff
@@ -59,3 +75,23 @@ def test_visit_count(self):
         result_data.increment_visit_count(coordinate1)
         self.assertEqual(3, result_data.get_visit_count(coordinate1))
         self.assertEqual(1, result_data.get_visit_count(coordinate2))
+
+    def test_measurement(self):
+        """
+        Test if CoordinateData can properly set and get the measurements.
+        """
+        coordinate_data = CoordinateData()
+
+        coordinate0 = Coordinate([0, 0, 0])
+        coordinate1 = Coordinate([0, 4, 1])
+
+        rcm0 = self._construct_rcm(10, 5, config_name="modelA_config_0")
+        rcm1 = self._construct_rcm(20, 8, config_name="modelB_config_0")
+
+        coordinate_data.set_measurement(coordinate0, rcm0)
+        measurement0 = coordinate_data.get_measurement(coordinate0)
+        self.assertEqual("modelA_config_0", measurement0.model_variants_name())
```

Comment:
this is only testing the name, and isn't actually testing anything about the measurement values?